source/includes/kubernetes_extension/_k8_serviceaccounts.md

```diff
@@ -0,0 +1,55 @@
+### Service accounts
+
+
+<!-------------------- LIST SERVICE ACCOUNTS -------------------->
+
+#### List service accounts
+
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/serviceaccounts"
```

Comment:
should there be a cluster_id parameter? I.e. "?cluster_id=a_cluster_id"
similar to the other GET calls in kubernetes_extension